src/entry.js

```diff
@@ -1,11 +1,13 @@
-const shared = require('./shared');
+const shared = require('./utils/shared');
 
 function StacheEntryPlugin() {
   const preload = (content, resourcePath, skyPagesConfig) => {
     try {
       const preloadPluginOrder = [
         require('./config'),
+        require('./json-data-element-attributes'),
         require('./include'),
+        require('./json-data-build-time'),
         require('./code-block'),
         require('./json-data'),
         require('./route-metadata'),
```

Comment:
It has to run after code block in order to avoid adding the elvis operator to all the escaped json-data in the code blocks.  It also has to run after the include plugin in order to translate and add the necessary data to the include blocks once they are added to the HTML. This was the key reason why the plugin ordering feature was necessary.